### PR TITLE
media-fonts/fontawesome-5.13.0: version bump + new proxy-maint

### PR DESCRIPTION
**Pull request by epsilon-0**
Close: https://bugs.gentoo.org/719036

Package-Manager: Portage-2.3.89, Repoman-2.3.20
Signed-off-by: Aisha Tammy <gentoo@aisha.cc>

### CODE REVIEW

File: media-fonts/fontawesome/fontawesome-5.13.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+MY_PN="Font-Awesome"
+inherit font
+
+DESCRIPTION="The iconic font"
+HOMEPAGE="https://fontawesome.com"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/FortAwesome/${MY_PN}.git"
+	KEYWORDS=""
```
Comment (juippis) [inline media-fonts/fontawesome/fontawesome-5.13.0.ebuild:14]: Please drop the KEYWORDS from this part altogether. It messes up with ekeyword tool.

File: media-fonts/fontawesome/fontawesome-5.13.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+else
+	SRC_URI="https://github.com/FortAwesome/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+	S="${WORKDIR}/${MY_PN}-${PV}"
+fi
+
+LICENSE="CC-BY-4.0 OFL-1.1"
+SLOT="0/5"
+IUSE="+otf +ttf"
```
Comment (juippis) [inline media-fonts/fontawesome/fontawesome-5.13.0.ebuild:23]: Are these the same files? I think you should prefer one over the other, not try to install duplicates.

File: media-fonts/fontawesome/fontawesome-5.13.0.ebuild (hunk 1 above)
Comment (epsilon-0) [reply media-fonts/fontawesome/fontawesome-5.13.0.ebuild]: Dropped :)

File: media-fonts/fontawesome/fontawesome-5.13.0.ebuild (hunk 2 above)
Comment (epsilon-0) [reply media-fonts/fontawesome/fontawesome-5.13.0.ebuild]: Changed to prefer otf over ttf by default :D .

**Pull request closed without merging**